frontend/src/scenes/insights/EmptyStates/EmptyStates.tsx

```diff
@@ -54,16 +54,18 @@ import { insightVizDataLogic } from '../insightVizDataLogic'
 export function InsightEmptyState({
     heading = 'There are no matching events for this query',
     detail = 'Try changing the date range, or pick another action, event or breakdown.',
+    icon = <IconArchive className="text-5xl mb-2 text-tertiary" />,
```

Comment:
I would recommend not adding a default one here just because this is used on many many places